src/main/scala/error/Error.scala

```diff
@@ -0,0 +1,5 @@
+sealed abstract case class Error(errorMessage: String) {
+  def getFormattedErrorMsg(): String = s"--> [${this.errorMessage}] !!!"
+}
+
+object InvaliInput extends Error("Invalid Input");
```

Comment:
```suggestion
object InvalidInput extends Error("Invalid Input")
```

occhio ai punti e virgola che non servono ;)